pybamm/input/parameters/lithium-ion/negative_electrodes/graphite_Yang2017/README.md

```diff
@@ -0,0 +1,12 @@
+# Graphite negative electrode parameters
```

Comment:
Are we using the Ecker parameter set for Yang? If so, we don't need to define it again. Otherwise, the Yang paper should be listed in the references here.